benchexec/tools/yogar-cbmc.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import benchexec.result as result
+import benchexec.util as util
+import benchexec.tools.template
+
+class Tool(benchexec.tools.template.BaseTool):
+
+    def executable(self):
+        return util.find_executable('yogar-cbmc')
+
+    def version(self, executable):
+        return '0.1'
+
+    def name(self):
+        return 'yogar-cbmc'
+
+    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
+        options += ['--no-unwinding-assertions']
+        options += ['--error-label', 'ERROR']
+        options += ['--mm', 'sc']
+        options += ['--32']
+        return [executable] + options + tasks
```

Comment:
Please do not hard-code parameters here, if they are not strictly always necessary. By adding them here, you would force these parameters onto all people that want to use BenchExec for benchmarking your tool.

In fact, adding `--m32` would probably make your tool behave wrong on all 64-bit benchmarks.

Furthermore, if you want to participate in SV-Comp with this tool, note that this tool-info module would also pass wrong parameters there (because in SV-Comp, you are not supposed to check for label `ERROR`).